docs/management/gcache-write-set-cache-encryption.md

```diff
@@ -0,0 +1,100 @@
+# GCache and Write-Set cache encryption
+
+The feature is a [tech preview](../glossary.md#tech-preview). Before using this feature in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+## GCache encryption Master Key
+
+GCache encrypts with a File Key. Creating the galera.gcache file or a full reset generates the File Key. The File Key is encrypted with the Master Key. 
+
+!!! admonition "See also"
+
+    For more information, see [Understanding GCache and Record-set Cache](../manual/gcache_record-set_cache_difference.md), and the Percona Database Performance Blog: [All you need to know about GCache](https://www.percona.com/blog/2016/11/16/all-you-need-to-know-about-gcache-galera-cache/)
+
+GCache has a RingBuffer on-disk file to manage write-sets. The file is pre-created when the server starts. This file contains a 1KB preamble to hold key-value pairs. The Master Key id and the encrypted File Key are stored in the GCache Ring preamble. The preamble is not encrypted because this section does not store sensitive data. 
```

Comment:
I'm not sure if we need this information about the preamble, its format, etc. This is the internal thing to Galera, user has nothing to set there (only curious user can look inside and see), but this could be a binary format of the preamble as well and it would have no inpact to the user.

Reply:
Removed.